ajax/libs/family.scss/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "family.scss",
+  "filename": "source/src/_family.scss",
+  "version": "1.0.8",
+  "description": "family.scss is a set of Sass mixins which will help you to manage the style of :nth-child'ified elements, in an easy way",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LukyVj/family.scss.git"
+  },
+  "keywords": [
+    "sass",
+    "scss",
+    "family",
+    "family.scss",
+    "nth-child",
+    "child",
+    "nth",
+    "eq",
+    "nth-of-type",
+    "nth-of-child",
+    "css",
+    "selector"
+  ],
+  "author": "Lucas Bonomi <@LukyVj> (http://lucasbonomi.com)",
+  "license": "ISC",
```

Comment:
I found that there has different [license](https://github.com/LukyVj/family.scss/blob/master/LICENSE) author mentioned.
Please discuss with author.
Thanks!